### PR TITLE
Add dropdown icon next to header and adds important information.

### PR DESCRIPTION
**Pull request by physi3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Thing Review
 
-An idea by me.
+An idea by me (@ollydevulder).
+
+With help from me (@physi3)
```
Comment (ollydevulder) [inline README.md:5]: These user reference are not links.
```suggestion
An idea by me ([@ollydevulder](https://github.com/ollydevulder)).

With help from me ([@physi3](https://github.com/physi3))
```

File: templates/base.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <link rel="icon" href="{% static 'icons/favicon.ico' %}">
     <title>{% block title %}Thing Review{% endblock %}</title>
 
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
```
Comment (ollydevulder) [inline templates/base.html:9]: I am reluctant to use external CSS.
1. It slightly increases load time (this is fairly insignificant)
2. Unnecessary CSS rules are loaded
3. See below :arrow_down:

File: templates/base.html [hunk 3]
```diff
@@ -14,7 +15,7 @@
 </head>
 <body>
     <div class="header">
-        <h1 onclick="toggleHeaderItems();">{% block header %}{% endblock %}</h1>
+        <h1 onclick="toggleHeaderItems();">{% block header %}{% endblock %}<i id="drop-down-icon" class="fa fa-fw fa-caret-down"></i></h1>
```
Comment (ollydevulder) [inline templates/base.html:18]: 3. (continued) It results in these ugly `class` attributes which are somewhat unclear

**Review by ollydevulder — CHANGES_REQUESTED** [review]
This is an effective solution for issue #10.
However, the use of external CSS is **not** encouraged.
A local solution would be preferred as it would be marginally better for performance and reliability.
Thanks :sparkles:

**Comment by ollydevulder** [comment]
This pull request contains the same commit that is requested #11. 
Please keep separate updates / fixes in different pull requests.
Thanks :sparkles:

File: templates/base.html (hunk 3 above)
Comment (physi3) [reply templates/base.html:18]: I thought you may say that and agree a local solution would be much neater. I will work on that.

**Pull request closed without merging**